src/index.js

```diff
@@ -4,41 +4,35 @@
 // An example of how you tell webpack to use a CSS (SCSS) file
 import './css/base.scss';
 
-// An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
-
-console.log('This is the JavaScript entry file - your code begins here.');
-
-console.log('proper clone test')
 
 import fetcher from './fetch.js';
+import User from './User.js';
 
-let userData, userTrips, tripsToDisplay, allDestinations;
+let userData, userTrips, tripsToDisplay, allDestinations, user;
```

Comment:
I don't believe `userDate` or `userTrip` are ever used in this class.

Reply:
I shall eliminate their existence.